LagCompensation/scripting/LagCompensation.sp

```diff
@@ -156,12 +157,23 @@ int g_aBlockTriggerMoved[MAX_EDICTS / 32];
 int g_aBlacklisted[MAX_EDICTS / 32];
 
 Handle g_hCookie_DisableLagComp;
+Handle g_hCookie_LagCompMessages;
 bool g_bDisableLagComp[MAXPLAYERS + 1];
+bool g_bLagCompMessages[MAXPLAYERS + 1];
 int g_iDisableLagComp[MAXPLAYERS + 1];
+ConVar g_cvCompenseEntityClass;
+ConVar g_cvCheckUserPing;
+ConVar g_cvMinimumPing;
 
 public void OnPluginStart()
 {
-	CreateConVar("sm_lagcomp_version", PLUGIN_VERSION, "LagCompensation Version", FCVAR_SPONLY|FCVAR_NOTIFY|FCVAR_DONTRECORD).SetString(PLUGIN_VERSION);
+	LoadTranslations("common.phrases");
+
+	g_cvCompenseEntityClass = CreateConVar("sm_lagcomp_entityclass", "0", "Only lagcompensate: \n0 = All \n1 = func_physbox only", _, true, 0.0, true, 1.0);
```

Comment:
i dont think that \n is a good usage in cvar description

Reply:
Fixed